includes/admin/customers/customers.php

```diff
@@ -355,6 +355,39 @@ function edd_customers_view( $customer ) {
 		</table>
 	</div>
 
+	<?php do_action( 'edd_customer_before_downloads', $customer ); ?>
+	
+	<div id="customer-downloads-wrapper" class="customer-section">
+		<h3><?php printf( __( 'Purchased %s', 'edd' ), edd_get_label_plural() ); ?></h3>
+		<?php
+			$downloads = edd_get_users_purchased_products( $customer->id );
```

Comment:
This needs to be `$customer->email` since you are getting `user's` purchased products, not `customer's`, and it's no guarantee that a `user_id` will be set in the case of guest purchases. The `email` will always exist.